instrumentation/jedis/jedis-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jedis/v3_0/JedisDbAttributesGetter.java

```diff
@@ -23,7 +30,7 @@ public String getDbNamespace(JedisRequest request) {
 
   @Override
   public String getDbQueryText(JedisRequest request) {
-    return request.getStatement();
+    return sanitizer.sanitize(request.getOperation(), request.getArgs());
```

Comment:
this was part of removing dependency on GlobalOpenTelemetry from inside `JedisRequest` without having to inject another field which would increase size of the (instantiated-at-runtime) `JedisRequest`